egs/fisher_swbd/s5/local/nnet3/run_tdnn_discriminative.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
```

Comment:
This script is fine other than possibly needing tuning for learning rate and num epochs.